praw/models/reddit/subreddit.py

```diff
@@ -378,7 +418,19 @@ def stream(self):
 
     @cachedproperty
     def stylesheet(self):
-        """Provide an instance of :class:`.SubredditStylesheet`."""
+        """Provide an instance of :class:`.SubredditStylesheet`.
+
+        For example, to add the css data ``.test{color:blue}`` to the existing
+        stylesheet:
+
+        .. code-block:: python
+
+            subreddit = reddit.subreddit('SUBREDDIT')
+            stylesheet = subreddit.stylesheet()
+            stylesheet += ".test{color:blue}"
```

Comment:
Should there be a preceding new line? Or does minimized CSS not require that?

Reply:
It does not